src/components/screens/monitor/transactions/index.js

```diff
@@ -6,14 +6,30 @@ import { transformStringDateToUnixTimestamp } from '@utils/datetime';
 import { toRawLsk } from '@utils/lsk';
 import Transactions from './transactions';
 
+const defaultTimestampRange = '1483228800000:1735689600000';
+const defaultAmountRange = '0:9223372036854775807';
```

Comment:
@reyraa would be really helpful if we can add comments as to why these particular values have been chosen as defaults.